client/pages/settings/workspace.vue

```diff
@@ -222,6 +331,42 @@ const customDomainsEnabled = computed(
   () => useRuntimeConfig().public.customDomainsEnabled,
 )
 
+const users = ref([])
```

Comment:
Same as above, users and loading state to update users should be in the dedicated table component